### PR TITLE
Update Batch supporting assemblies to target new SDK version

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

**Comment by matthchr** [comment]
@weshaggard, @chidozieononiwu - What's the standard operating procedure for PRs? Should I be tagging certain folks or pinging certain folks, or you'll get to them in a few days if I just let it sit?

**Comment by weshaggard** [comment]
@mattchr that is a great question I think for now at least it would be good if you could ping @AlexGhiondea and myself, it would also be good to get sign-off from someone from your team. 

@mattchr do you have access to merge?

**Review by weshaggard — APPROVED** [review]

**Comment by matthchr** [comment]
@weshaggard - Yes it looks like once you have approved I can merge. Let me ask one of my coworkers to take a quick look as well.

@xingwu1, @bgklein - one of you wanna take a look?

**Review by bgklein — APPROVED** [review]

**Pull request merged**
